decadog_core/src/lib.rs

```diff
@@ -206,4 +221,16 @@ impl<'a> Client<'a> {
     pub fn get_members(&self) -> Result<Vec<OrganisationMember>, Error> {
         self.github.get_members(self.owner)
     }
+
+    /// Update milestone title with provided title
+    pub fn update_milestone_title(
+        &self,
+        milestone: &Milestone,
+        new_title: String,
+    ) -> Result<Milestone, Error> {
+        let mut update = MilestoneUpdate::default();
+        update.title = Some(new_title);
```

Comment:
As the update only has one field, you can just `let update = MilestoneUpdate { title: Some(new_title) };`